sass/base/components/email-signup/_email-signup.scss

```diff
@@ -781,8 +781,28 @@ select,
       @include span(10);
     }
   }
+
+}
+
+// ESU Workplace remove margin top to privacy message on medium screen
+@include breakpoint($screen-md) {
+  .cr-email-signup-workplace-form {
+    .esu-privacy-message {
+      margin-top: -25px;
+    }
+  }
```

Comment:
I think we might  want to look at why this needs to be up with a negative margin here. It appears the form on the workplace ESU has a bottom-padding that is probably not necessary (anymore). 

Also I think we might want to consider putting the privacy message code inside the already targeted ESUs rather than targeting each ESU variant again here, but something we can refactor in a different ticket.